forta-uniswap-event-py/src/agent.py

```diff
@@ -0,0 +1,216 @@
+import binascii
+import json
+
+from forta_agent import Finding, FindingType, FindingSeverity
+from web3 import Web3
+import web3
+
+# Uniswap v2: Router 2
+# 0x7a250d5630b4cf539739df2c5dacb4c659f2488d
+ROUTER_ADDR = Web3.toChecksumAddress("0x7a250d5630b4cf539739df2c5dacb4c659f2488d")
+
+ETHER_THRESHOLD = Web3.toWei("5", "ether")
```

Comment:
For our agent suites, we have been breaking out our alert threshold values into a separate `agent-config.json` file.  It may be worthwhile to follow that convention/pattern for this agent as well.